WordPressVIPMinimum/Tests/Filters/RestrictedFilterUnitTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Unit test class for WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Tests\Filters;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+/**
+ * Unit test class for the Filters/AlwaysReturn sniff.
+ *
```

Comment:
Add a `@since` version number for whatever the next version will be (presumably `0.4.0`).